### DISCUSSION TITLE
Support for STM32L071

### DISCUSSION THREAD

**Discussion by bilbo69** [stm32duino/Arduino_Core_STM32#1878] [Q&A] [answered]
Hello,
I have a project where I would like to use a STM32L071CBT6.
I noticed that issue 1129 is closed. It provides compatibility with STM32L071 versions.
In the list of versions, in the Arduino IDE, however, I don't see the L071s but only some L072s.
Does anyone know how to fix the problem?

![image](https://user-images.githubusercontent.com/45975719/203616302-1b6ab667-2356-4693-98ad-7ac44cf1b7b3.png)

**Comment by fpistm** [comment, marked as answer]
Hi
No #1129 provides support of real eeprom of stm32l0 not all mcu. 
L071 is not available anyway you can add it following wiki.
